### PR TITLE
0.3.1 release

### PR DESCRIPTION
**Pull request by Evolution0**
Updated requirements and syntax to work in newer versions of Atom

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "language-geck",
-  "version": "0.3.0",
+  "version": "0.3.1",
   "description": "Syntax highlighting for Fallout 3/NV scripting",
-  "repository": "https://github.com/nyteshade/language-geck.git",
```
Comment (nyteshade) [inline package.json]: Please reset the repository and I'm good to go!

**Comment by nyteshade** [comment]
Thanks for doing this. I was meaning to get around to it. The only issue I have is that the repository field has been changed to yours. Please fix it to mine and I'll happily merge the fixes.

**Comment by Evolution0** [comment]
Woops, originally wasn't going to do a PR as I thought the repo was dead but changed my mind last minute (explains the repo), I'll fix that.

Edit: *cough* ignore the typo..

**Comment by nyteshade** [comment]
Thanks for the fix and the PR. Merged.

**Pull request merged**